src/skins/blau.tsx

```diff
@@ -92,10 +92,14 @@ export const getBlauSkin: GetKnownSkin = () => {
             buttonPrimaryBackgroundHover: palette.blauBlueSecondary60,
             buttonPrimaryBackgroundSelectedInverse: palette.blauBluePrimary30,
 
-            buttonSecondaryBackground: palette.blauBlueSecondary,
-            buttonSecondaryBackgroundSelected: palette.blauBlueSecondary60,
+            buttonSecondaryBorder: palette.blauBlueSecondary,
+            buttonSecondaryBorderSelected: palette.blauBlueSecondary60,
             buttonSecondaryBorderInverse: palette.white,
-            buttonSecondaryBorderSelectedInverse: palette.blauBluePrimary30,
+            buttonSecondaryBorderSelectedInverse: palette.white,
+            buttonSecondaryBackgroundHover: palette.blauBlueSecondary10,
```

Comment:
this color is not used?

Reply:
true, buttonSecondaryBackgroundHover was missing in button css in hover state, changed.

now, all background styles is using constants with "background" name and borders is using "border" in constant name

`buttonSecondaryBorderSelectedInverse` is being used, right?

Reply:
hmmm, yes, but I don't see usages of `buttonSecondaryBackgroundSelectedInverse`

Reply:
fixed